airbnb_project/listings/listing_models.py

```diff
@@ -1,16 +1,19 @@
 from django.db import models
 
 class Listing(models.Model):
-  name = models.CharField(max_length=100)
-  baths = models.CharField(max_length=10)
-  beds = models.CharField(max_length=10)
-  latitude = models.CharField(max_length=20)
-  location = models.CharField(max_length=20)
-  longitude = models.CharField(max_length=20)
-  person_capacity = models.CharField(max_length=20)
-  registration_number = models.CharField(max_length=20)
-  room_type = models.CharField(max_length=20)
-  title = models.CharField(max_length=100)
+    airbnb_listing_id = models.TextField(blank=False, null=False)
```

Comment:
Are these model definitions aligned with the `listing_processor`? Is there any field that the `listing_processor` would need that we are possibly considering optional here?

Reply:
For the processer, I think the `registration_number` is the only one that is needed, @nam-m can you comment on this?

Reply:
We might want to establish this 'contract' with AI team.

Reply:
> For the processer, I think the `registration_number` is the only one that is needed, @nam-m can you comment on this?

sorry i missed this, yes my part only needs `registration_number` for now